bin/compare/compare-script.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+BEFORE_BRANCH=$1
+AFTER_BRANCH=$2
+
+git checkout $BEFORE_BRANCH > /dev/null
+npm install > /dev/null
```

Comment:
maybe use `npm ci` instead here? That ensures node_modules is clear and just `package-lock.json` is used which should prevent a new package-lock.json from being created.